data/csp/sap-converged-cloud/sle15/preferences.yaml

```diff
@@ -0,0 +1,18 @@
+preferences:
+  type:
+    _attributes:
+      devicepersistency: Null
+      format: vmdk
+      firmware: uefi
+      kernelcmdline:
+        console: ["ttyS0,115200n8", "tty0"]
+        NON_PERSISTENT_DEVICE_NAMES: 1
+        multipath: "off"
+      vga: normal
+      machine:
+        _attributes:
+          ovftype: vmware
+          HWversion: 10
+        vmdisk:
+          _attributes:
+            diskmode: streamOptimized
```

Comment:
`machine` is a child of `type` not an attribute. Just move it one indentation level to the left.

Reply:
Done.